config/tracks/ks_nordschleife.ini

```diff
@@ -1221,15 +1328,6 @@ KEY_0=ksEmissive
 VALUE_0=164,164,255,0.006
 VALUE_0_OFF=0
 
-[MATERIAL_ADJUSTMENT_31]
-ACTIVE=1
-VISIBILITY_LEVEL=0
-DESCRIPTION=Dorint Hotel windows
-MATERIALS=MATdorintwin?
-KEY_0=ksEmissive
-VALUE_0=255,255,255,0.006
-VALUE_0_OFF=0
-
```

Comment:
Removed material_adjustment_31 as it is redundant with material_adjustment_33 and serves no purpose.